src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -99,49 +161,75 @@ public function getHorizontalBorderChar()
      * @param string $verticalBorderChar
      *
      * @return $this
+     *
+     * @deprecated since Symfony 4.1. Use {@link setVerticalBorderChars()} instead.
      */
     public function setVerticalBorderChar($verticalBorderChar)
     {
-        $this->verticalBorderChar = $verticalBorderChar;
+        @trigger_error(sprintf('Method %s() is deprecated since Symfony 4.1. Use setVerticalBorderChars() instead.', __METHOD__), E_USER_DEPRECATED);
 
-        return $this;
+        return $this->setVerticalBorderChars($verticalBorderChar, $verticalBorderChar);
     }
 
     /**
      * Gets vertical border character.
      *
      * @return string
+     *
+     * @deprecated since Symfony 4.1. Use {@link getBorderChars()} instead.
      */
     public function getVerticalBorderChar()
     {
-        return $this->verticalBorderChar;
+        @trigger_error(sprintf('Method %s() is deprecated since Symfony 4.1. Use getBorderChars() instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
same here, use a comma instead of a dot.

Reply:
Fixed.